.github/workflows/dns_entry_checker.yaml

```
@@ -0,0 +1,33 @@
+name: DNS Entry Checker Unittest
+
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
+    paths:
+      - "dns_entry_checker/**"
+      - ".github/workflows/dns_entry_checker.yaml"
+
+jobs:
+  test_and_lint:
```

As mentioned in #77 you're not actually doing any linting 

```suggestion
  test_with_unittest:
```
You can add this to make github actions perform linting

```
    - name: Analyse with pylint 
      run: |
          pylint --recursive yes test dns_entry_checker
```
you'll also need to put a .pylintrc file in dns_entry_checker folder